package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "pdap-design-system",
-	"version": "2.0.1",
+	"version": "2.0.2",
```

Comment:
Since we already published 2.0.2, do we want to bump the version in a standalone PR for the sake of git history?